File: projectGood/DAL/Models/OrderTicket.cs
using System;
using System.Collections.Generic;

namespace DAL.Models;

public partial class OrderTicket
{
    public short OrdId { get; set; }

    public short? CustId { get; set; }

    public DateTime? OrdDate { get; set; }

    public TimeSpan? OrdTime { get; set; }

    public short? TripId { get; set; }

    public int? CountPlaces { get; set; }

    public virtual Customer? Cust { get; set; }

    public virtual Trip? Trip { get; set; }
}

File: angular_project/src/app/classes/OrderTicket.ts
import { Time } from "@angular/common";

export class OrderTicket{
    [x: string]: any;
    constructor(
        public ordId?:number,
        public custId?:number,
        public ordDate?:Date,
        public ordTime?:Time,
        public tripId?:number,
        public countPlaces?:number,
        public customerFullName?:string,
        public destination?:string,
        public tripDate?:Date   
        ){

    }
}